packages/api/src/user/user.ts

```diff
@@ -113,13 +114,15 @@ export const getPlatformUserBalance = async (
   call: Call,
   { userType, platformUserId, tokenId }: PlatformUserTokenBalancesArgs,
 ) => {
-  const response = await call<Response<{ token: string }>>({
+  const response = await call<Response<{ token: TokenSymbolType }>>({
```

Comment:
Hey @nsstarr the problem as @teimurjan already mentioned is over here... We are returning the wrong response type the correct one should be like the one Tim sent you `{ token, amount, createdAT }`. Then you are going to be able to use the correct typed values on the example and print them nicely using printable.

Reply:
Thanks, @diegonzs-roll . I've updated the response type and changed the formatting in the printTable. Everything now prints out nicely.